devops/ansible/roles/girder/tasks/npm-RedHat.yml

```diff
@@ -1,23 +1,19 @@
 ---
 
-- name: NodeJS | Download repo setup script
-  # Due to issues with this potentially running on python < 2.7.9, validating
-  # certs via Python (ansible) can be an issue.
-  # We could use curl and bypass this, but instead we'll just disable cert validation
-  # and require the output match the known good checksum.
-  get_url:
-    url: https://rpm.nodesource.com/setup_4.x
-    dest: "/tmp/setup_4.x"
-    validate_certs: no
-    checksum: sha256:3359378636868f9fd9f0565a28191ed7e831e91b703b581ab9ff737a4574e5f4
-
 - name: NodeJS | Install repo
-  shell: "bash /tmp/setup_4.x"
+  yum_repository:
```

Comment:
On a fresh RHEL7 instance in AWS EC2 (which has PYthon 2.7.5), I ran:
```
sudo yum update
sudo yum install gcc python-devel openssl-devel
curl -O https://bootstrap.pypa.io/get-pip.py
sudo python get-pip.py
sudo pip install ansible
ansible-playbook npm-RedHat.yml
```
Where I modified the `npm-RedHat.yml` to make it into a standalone playbook by adding a
```
- hosts: localhost
  tasks:
    - ...
```
to the top. This served to install NodeJS v6.10.2.

So, this provisions from RHEL 7 onto RHEL 7 with no problems.